Mathlib/AlgebraicGeometry/Pullbacks.lean

```diff
@@ -619,7 +619,7 @@ variable (R S T : Type u) [CommRing R] [CommRing S] [CommRing T] [Algebra R S] [
 
 open TensorProduct Algebra.TensorProduct CommRingCat RingHomClass
 
-/-- The isomorphism between the fiber product of two schemes `Spec S` and `Spec T`
+/-- The isomorphism between the fibre product of two schemes `Spec S` and `Spec T`
```

Comment:
Actually, should I change this to `fibred` to align with the rest of the file which uses "fibred product"?

Reply:
Sounds good to me.

Reply:
```suggestion
/-- The isomorphism between the fibred product of two schemes `Spec S` and `Spec T`
```